src/AddTodoForm.js

```diff
@@ -1,14 +1,22 @@
 import React from "react";
 
 
-function AddTodoForm(){
+function AddTodoForm(props){
+
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        let todoTitle = event.target.title.value;
```

Comment:
This isn't wrong, but we never change the value of `todoTitle`, so it would be better for it to be a `const`.